spectrum-offchain/src/backlog.rs

```diff
@@ -368,34 +372,70 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn should_suspend_existing_order() {
-        let mut backlog = setup_backlog(10, 5, 50);
+    async fn should_suspend_existing_order_mock_store() {
+        should_suspend_existing_order(MockBacklogStore::new()).await
+    }
+
+    #[tokio::test]
+    async fn should_suspend_existing_order_rocksdb() {
```

Comment:
What's the point in duplicating those tests depending on Store type? Those tests test Backlog logic which should not depend on BacklogStore implementation at all.

Reply:
Wanted to transitively test the RocksDB implementation but I see your point. Removed. Adding separate test for RocksDB BacklogStore.